Code/Editor/EditorEngineProcessFramework/EngineProcess/EngineProcessMessages.h

```diff
@@ -299,14 +299,13 @@ class EZ_EDITORENGINEPROCESSFRAMEWORK_DLL ezExportDocumentMsgToEngine : public e
 
 public:
   ezExportDocumentMsgToEngine()
```

Comment:
Is there also a check that can remove empty constructors ?

Reply:
even empty costructors should not be removed but instead be ` = default` in the cpp file so that the linker doesn't have to deduplicate them.